### PR TITLE
QuasiQuoters for KORE terms

### PR DESCRIPTION
**Pull request by goodlyrottenapple**
Part of broader efforts to improve tests in the booster (#186)
Add trm and symb quasiquoters
Convert some unit tests to use the QQs

### CODE REVIEW

**Review by jberthold — APPROVED** [review]
LGTM , nice work.
Although the lines are getting rather long :disappointed: ...

**Review by jberthold — APPROVED** [review]
code LGTM, I updated the branch to latest `main` after merging two other open PRs to unblock it from the `cabal` issue with `yaml`.
Briefly looked into avoiding the `syb` dependency but could not, and it is not a big deal.
@geo2a ?

**Review by geo2a — APPROVED** [review]

**Pull request merged**
